### ISSUE TITLE
Selected Editors / Reviewers not persisting data

### ISSUE THREAD

**Issue by Cooryd** [elifesciences/elife-xpub#2170]
**Describe the bug**

It doesn't look like we are sending and persisting selected reviewer and editor information from the submission wizard any more.

**Steps To Reproduce**

- Create a new submission, navigate through to the the Editors step.
- Select 2 Editors ands 2 Reviewers then move through to the Disclaimer step
- Fill in acknowledgment and consent tick box. 
- Wait a few second (5) to ensure auto save has picked up all form changes and persisted them
- refresh page and navigate to the Editors step, no editors / reviewers will be visible
- Checking the disclosure steps fields have persisted confirms auto save did occur

**Expected behavior**

Editors and reviewers should persist.

**Screenshots**

**Device, Operating System and Browser**

OSX, Chrome

**Comment by Cooryd** [comment]
~Looks like the data __IS__ being sent over on auto save and populating DB. Could be fetch data issue or initial data of form not being correct.~ Team is saving, no member data is being stored however

**Comment by Cooryd** [comment]
Correct data is present up until `.save()` on the `manuscript` object 

https://github.com/elifesciences/elife-xpub/blob/541e0ac496a53127d7f4a5661f86157039708b81/packages/component-submission/server/aggregates/Submission.js#L68

Attempted replacing `Manuscript` save with a direct save on `Team` entity and issue still occurs. Seems to be that the new data isn't being passed to postgres, can only see old data appearing in postgres logs. This looks like an issue with our `Objection` related save.

**Comment by Cooryd** [comment]
Tried getting `_saveTeams()` working with `Team` objects instead of `Manuscript` to fix issue with relations. Same problem occurs so looks like this isn't a relation issue. 

```
  async _saveTeams() {
    await this.teams.forEach(async team => 
      await team.save()
    )
  }
```

**Comment by will-byrne** [comment]
this was introduced in #2140 
commit: 1c7f0cd56369cd744a2bc48021972b043f5a5187

**Comment by Cooryd** [comment]
Closed, fixed with PR #2173
